libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java

```diff
@@ -32,13 +32,30 @@
 
 public class SystemInformation {
   private static final Log logger = LogFactory.getLog( SystemInformation.class );
+  private static LinkedHashMap<String, String> map;
+
+  protected static LinkedHashMap<String, String> getMap() {
+    return map;
+  }
+
+  protected static void setMap( LinkedHashMap<String, String> map ) {
+    SystemInformation.map = map;
+  }
+
+  public Log getLogger() {
+    return logger;
+  }
 
   public SystemInformation() {
   }
 
   public void logSystemInformation() {
-    if ( logger.isInfoEnabled() ) {
-      logger.info( getSystemInformation() );
+    if ( getLogger().isInfoEnabled() ) {
+      getLogger().info( getOtherProperties() );
```

Comment:
Yes it should! Sorry for the slip-up. I must have changed it inadvertently during the refactoring. I'll fix it.